seacatauth/client/service.py

```diff
@@ -15,9 +15,9 @@
 
 #
 
-# TODO: Implement support for remaining metadata
-# TODO: Implement support for other response and grant types and stuff
 # https://openid.net/specs/openid-connect-registration-1_0.html#ClientMetadata
+# TODO: This is not Client responsibility!
```

Comment:
What is "This" here?

Reply:
this line is probably not necessary, i will delete it. the following line explains it clearly.

Reply:
ok